_layouts/home.html

```diff
@@ -31,5 +30,13 @@ <h1 class="page-heading center">Contributors</h1>
         <i class="fa fa-facebook"></i>
       </a>
     </li>
+    <li>
+      <a href="https://github.com/ashunimbz"> 
+        Ashish Nimbalkar
+      </a>
+      <a href="https://facebook/ashunimbz">
```

Comment:
Hey @ashunimbz  - The link for facebook is missing the TLD, `.com` that is. 
Could you add that? 
Also, just make a new commit on top of this same PR rather than submitting a new one.

Reply:
yes sure.
Fixed and made a new commit .